src/main/frontend/components/content.cljs

```diff
@@ -170,9 +169,10 @@
           [:div.flex-row.flex.justify-between
            (for [color block-background-colors]
              [:a.m-2.shadow-sm
-              {:on-click (fn [_e]
-                           (editor-handler/set-block-property! block-id "background-color" color))}
-              [:div.heading-bg {:style {:background-color color}}]])
+              {:title (t (keyword "color" color))
+               :on-click (fn [_e]
+                           (editor-handler/set-block-property! block-id "background-color" (str "var(--ls-highlight-color-" color ")")))}
+              [:div.heading-bg {:style {:background-color (str "var(--color-" color "-500)")}}]])
```

Comment:
We are using a naming convention based on the `block-background-colors` vector.
A vector of maps that explicitly sets the name of the color and the css properties could be safer and more open, but it feels a little verbose.
```
(def block-background-colors
   [{:id "gray"
     :title (t :color/gray)
     :default-color "var(--color-gray-500)" 
     :highlight-color "var(--ls-highlight-color-gray)"}
   ...])